### ISSUE TITLE
Station name is not working

### ISSUE THREAD

**Issue by p051xn1nja** [gpailler/MMM-aqicn#9]
Hi there,

I tried all kind of combinations for the name of the station, including city and country but its not working. I am a little bit frustrated. Please advice...

**Comment by gpailler** [comment]
Hi,

Did you use the following website to test your token and identify the exact station name you should use?
http://aqicn.org/json-api/demo/

**Comment by p051xn1nja** [comment]
I used this name in this link

https://aqicn.org/city/romania/arges/pitesti/str.-nicolae-balcescu/

I mean this name

Str. Nicolae Bălcescu, Piteşti

And is not working

I verified the token and is working

**Comment by gpailler** [comment]
There is maybe an issue with non ASCII characters.
Does it work with the sample provided: `Bangkok University Rangsit Campus, Pathum`
Does it work with something like: `Nicolae Balcescu`

**Comment by p051xn1nja** [comment]
No, it's not working with the default station. And I used only Latin
standard without ă, just a and is not working.

**Comment by gpailler** [comment]
To investigate further, you should open your MagicMirror interface from a classic browser and press CTRL-SHIFT-i to display the Developer tools. Then you can probably see some error message in the Console

**Comment by p051xn1nja** [comment]
Now, its working.

**Comment by gpailler** [comment]
Ok great! Did you change something or was it a transient issue?

**Comment by turntoblue** [comment]
I have the same issue. What did you change?

**Comment by turntoblue** [comment]
See below config,
{
        module: "MMM-aqicn",
        position: "top_right",
        header: "Air Quality Index",
        config: {
            token: "f1e46a2c066ab052a24297024538ad6b60c2563a",
            stationName: "Gugal, South Korea",
            showDetails: true
            }
        },

The Module started without issue but it just showed that "station name not found"

**Comment by icarus323** [comment]
I confirmed that my token and station name are displayed correctly by entering them on the website. 
However, an error message saying that "station name not found" is displayed. 
I think the station name should consist only of alphabets to function properly. 
My station is displayed as "Gyeongchun-ro, Gyeonggi, South Korea (경춘로 경기)" on the aqicn website. 
If you change "stationName" to "city id", it should work properly."
